### PR TITLE
add new UI

### PR DESCRIPTION
**Pull request by PedroH-Neves**
Here I created the design of the calculator using react.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (MeqdamAlqudah) [inline src/components/Calculator.js]: - [ ] Please the Calculator.js component should be a functional component.

- [ ] Please `Calculator.js` should have at least 1 child component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (MeqdamAlqudah) [inline src/components/Calculator.js:5]: - [ ] Please your code should match the design(the buttons colors , the output box background color).
Your code 
image
Design 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @PedroH-Neves 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image
## Highlights
:+1: usage of react 
:+1: components

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Display from './Display';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (AnselemOdims) [inline src/components/Calculator.js]: - [ ] As pointed out by the previous reviewer, please kindly ensure that the `Calculator` component is a functional component as opposed to a class component.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @PedroH-Neves , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @PedroH-Neves 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 🥇  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [Branch name](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) should have a name of a feature or action related to your project 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**